manifests/install.pp

```diff
@@ -48,4 +48,10 @@
       ensure => installed,
     }
   }
+
+  if $::foreman::rails_cache_store['type'] == 'redis' {
```

Comment:
If type is not present in the hash, I think this will fail and the type doesn't enforce it currently.